### ISSUE TITLE
Missing content nodes in GQL data after migrating to this plugin

### ISSUE THREAD

**Issue by davehulse** [angeloashmore/gatsby-source-prismic#313]
I have been migrating a site from using the gatsby-source-prismic-graphql to gatsby-source-prismic.

I noticed after configuring the new plugin that certain nodes in my GQL queries were coming up as empty. This was after creating the matching json schemas that are correctly named and linked to in the config (using the exact name as the type name in Prismic).

for example:
```
{
  resolve: 'gatsby-source-prismic',
    options: {
      repositoryName: 'XXXXXX',
      accessToken: 'XXXXXX'
      schemas: {
        cookie_notice: require('./src/schemas/cookie_notice.json'),
        footer: require('./src/schemas/footer.json'),
        header: require('./src/schemas/header.json'),
        homepage: require('./src/schemas/homepage.json'),
        secondary_page: require('./src/schemas/secondary_page.json'),
        tertiary_page: require('./src/schemas/tertiary_page.json')
      },
      linkResolver: () => (doc) => linkResolver(doc)
    }
 }
```
As you can see from the screenshot below, the Prismic GQL playground displays the correct "totalCount" numbers for each content type (which matches the number of content nodes for that content type). But in my localhost version (with converted GQL naming format) I am getting either empty totalCounts or less than actually exists in Prismic. I doesn't seem to have a different effect whether it's a single vs repeatable content type.

Any ideas why this could be happening?
![graphqlcomparison](https://user-images.githubusercontent.com/45797696/106820486-4068b780-6630-11eb-883f-f7ab21f0d33a.png)

**Comment by angeloashmore** [comment]
Hi @davehulse, I see that you closed this issue. Was this resolved on your end?

**Comment by davehulse** [comment]
Thanks for following up @angeloashmore

I ended up taking a cue from Prismic's own gatsby-blog repo:
https://github.com/prismicio/gatsby-blog

I moved my content type schemas from my previous /src/schemas/ directory into a /custom_types/ folder at the root of the project. I updated the config urls to point that folder instead, such as:

`
secondary_page: require('./custom_types/secondary_page.json'),
(etc...)
`
I ran the build again, and it pulled in all the data correctly. My previous paths were seemingly pointing to the right place, so not sure why this made a difference, but it worked.
